signals/parser/schema.py

```diff
@@ -28,6 +29,25 @@ def create_objects(self, object_json):
             data_object = DataObject(object_name, data_object_json[object_name])
             self.data_objects[data_object.name] = data_object
 
+    # Note: Why do we call them "apis" in method names, but then call them "urls" in the schema/code
+    def validate_apis_and_objects(self):
+        for url in self.urls:
+            for endpoint in URL.URL_ENDPOINTS.keys():
+                api = getattr(url, endpoint, None)
+                if api:
+                    if getattr(api, 'parameters_object', None):
```

Comment:
I wanted to do that, but if I write `if api['parameters_object'] not in self.data_objects:` I get an error.